CorePreferences/src/au/gov/asd/tac/constellation/preferences/GraphPreferenceKeys.java

```diff
@@ -46,6 +48,13 @@ public final class GraphPreferenceKeys {
     //
     public static final String LEFT_COLOR_DEFAULT = "Green";
     public static final String RIGHT_COLOR_DEFAULT = "Magenta";
+    
+    public static final String ENABLE_ANIMATIONS = "enableAnimations";
+    public static final boolean ENABLE_ANIMATIONS_DEFAULT = true;
+    
+    public static boolean isAnimatable (final Preferences prefs) {
+        return prefs.getBoolean(ENABLE_ANIMATIONS, ENABLE_ANIMATIONS_DEFAULT);
+    }

```

Comment:
Is there another place we can put this function? Generally I think it would be good to leave the preference keys classes to contain just the keys themselves